addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4689,6 +4701,9 @@ registry.sizing = SnippetOptionWidget.extend({
                 }, 0);
 
                 self.options.wysiwyg.odooEditor.automaticStepActive('resizing');
```

Comment:
I am surprised that this is not called if we reach the `return` just above... is that right? In any case I think it would be better to free the mutex right after this (and if it happens that this should indeed be after the `return`, you would obviously need to free the mutex before the return too).

Reply:
It's not probably not required. But from a consistency point of view, I would say "Ok the action is finished" (= freeing the edition mutex) at the very end.